resources/views/partials/bootstrap-table.blade.php

```diff
@@ -44,87 +44,44 @@ classes: 'table table-responsive table-no-bordered',
-            sidePagination: '{{ (isset($clientSearch)) ? 'client' : 'server' }}',
-            sortable: true,
-            @if (!isset($nopages))
-            pageSize: 20,
-            pagination: true,
-            @endif
             cookie: true,
             cookieExpire: '2y',
             cookieIdTable: '{{ Route::currentRouteName() }}',
-            @if (isset($columns))
-             columns: {!! $columns !!},
-            @endif
             mobileResponsive: true,
             maintainSelected: true,
+            trimOnSearch: false,
             paginationFirstText: "{{ trans('general.first') }}",
             paginationLastText: "{{ trans('general.last') }}",
             paginationPreText: "{{ trans('general.previous') }}",
             paginationNextText: "{{ trans('general.next') }}",
+            pageList: ['20', '30','50','100','150','200', '500'],
+            paginationVAlign: 'both',
             formatLoadingMessage: function () {
                 return '<h4><i class="fa fa-spinner fa-spin" aria-hidden="true"></i> Loading... please wait.... </h4>';
             },
+
             icons: {
                 advancedSearchIcon: 'fa fa-search-plus',
                 paginationSwitchDown: 'fa-caret-square-o-down',
                 paginationSwitchUp: 'fa-caret-square-o-up',
                 columns: 'fa-columns',
-                @if (isset($multiSort))
-                sort: 'fa fa-sort-amount-desc',
-                plus: 'fa fa-plus',
-                minus: 'fa fa-minus',
-                @endif
                 refresh: 'fa-refresh'
             },
-            exportDataType: 'all',
             exportTypes: ['csv', 'excel', 'doc', 'txt','json', 'xml', 'pdf'],
-            exportOptions: {
-
-                fileName: '{{ ((isset($exportFile)) ? $exportFile : 'table') }}-' + (new Date()).toISOString().slice(0,10),
-                ignoreColumn: ['actions','image','change','checkbox','checkincheckout','icon'],
-                worksheetName: "Snipe-IT Export",
-                escape: false,
-                jspdf: {
-                    orientation: 'l',
-                    autotable: {
-                        styles: {
-                            rowHeight: 20,
-                            fontSize: 10,
-                            overflow: 'linebreak',
-                        },
-                        headerStyles: {fillColor: 255, textColor: 0},
-                        //alternateRowStyles: {fillColor: [60, 69, 79], textColor: 255}
-                    }
-                }
-            },
-
-            @if (!isset($simple_view))
-
-                showRefresh: true,
-                stickyHeader: true,
-                stickyHeaderOffsetY: stickyHeaderOffsetY + 'px',
-
-                @if (isset($showFooter))
-                showFooter: true,
-                @endif
 
-                showColumns: true,
-                trimOnSearch: false,
 
+            @if (!isset($simple_view))
```

Comment:
I would rather see this as something like - 

```javascript
{
 /*  ... */
  showMultiSort: {{ !isset($imple_view) && isset($multiSort) }}
}
```

Because I get weird feelings when I see lots of conditional stuff in a view that dynamically changes javascript. It just freaks me out and it hard to mentally parse and map between multiple different languages.

Reply:
I'm honestly not even sure that we use that option anymore.